internal/object.go

```diff
@@ -165,3 +172,51 @@ func setObject(ctx context.Context, doc *enigma.Doc, info *enigma.NxInfo, props
 	}
 	return nil
 }
+
+func Publish(ctx context.Context, doc *enigma.Doc, objectID string) error {
+	object, err := doc.GetObject(ctx, objectID)
+	if err != nil {
+		return err
+	}
+	if object.Handle != 0 {
+		var layout Layout
+		var raw json.RawMessage
+		raw, err = object.GetLayoutRaw(ctx)
```

Comment:
nit: You could write
```
raw, err := object.GetLayoutRaw(ctx)
```
That way you don't have to declare the `raw` variable on the line above.

Reply:
interesting...I'm just used to declaring everything. BUf you are saying you don't have to do that because Go can autoassign a type to a variable?

Reply:
Yes, it can implicitly assign types to variables.

Reply:
But to be clear, there's nothing wrong with the way you wrote it. I'm only saying it can be written more tersely.